### PR TITLE
EAMT 2018 ingestion

### PR DESCRIPTION
**Pull request by davidstap**
Ingested EAMT 2018.

### CODE REVIEW

**Comment by mjpost** [comment]
Note we have a bunch of code-formatting tools in place since you last contributed. The new target `make autofix` might be helpful here.

**Comment by mjpost** [comment]
Is this ready then?

**Comment by mjpost** [comment]
(Also please merge in `origin/master`)

File: data/xml/2018.eamt.xml [hunk 1]
```diff
@@ -0,0 +1,529 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<collection id="2018.eamt">
+  <volume id="main" ingest-date="2023-05-05">
+    <meta>
+      <booktitle>Proceedings of the 21st Annual conference of the European Association for Machine Translation</booktitle>
```
Comment (mjpost) [inline data/xml/2018.eamt.xml:5]: "Conference" should be capitalized. Actually, I see that it is lowercased on [many other EAMTs](https://aclanthology.org/venues/eamt/)—can you fix those in this PR?

File: data/xml/2018.eamt.xml [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+      <bibkey>siarhei-bratchenia-2018-implementing</bibkey>
+    </paper>
+    <paper id="32">
+      <title>Bootstrapping Multilingual Intent Models via Machine Translation for Dialog Automation</title>
+      <author><first>Nicholas Ruiz and Srinivas Bangalore and John</first><last>Chen</last></author>
```
Comment (mjpost) [inline data/xml/2018.eamt.xml:289]: Name is incorrect. This is true for many names throughout the proceedings.

File: data/xml/2018.eamt.xml [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+      <author><first>Jan and Kris Luyten and Karin</first><last>Coninx</last></author>
+      <pages>387</pages>
+      <url hash="da892903">2018.eamt-main.57</url>
+      <abstract>We present the highlights of the now finished 4-year SCATE project. It was completed in February 2018 and funded by the Flemish Government IWT-SBO, project No. 130041.1</abstract>
+      <bibkey>van-eynde-etal-2018-smart</bibkey>
+    </paper>
+    <paper id="58">
+      <title>news.bridge - Automated Transcription and Translation for News</title>
+      <author><first>van</first><last>der Kreeft</last></author>
+      <author><first>Peggy and Renars</first><last>Liepins</last></author>
```
Comment (mjpost) [inline data/xml/2018.eamt.xml:514]: names

**Review by mjpost — COMMENTED** [review]
Names are incorrectly parsed throughout. Please go through the [ingestion checklist](https://github.com/acl-org/acl-anthology/wiki/Ingestion-Checklist).

File: data/xml/2018.eamt.xml (hunk 1 above)
Comment (davidstap) [reply data/xml/2018.eamt.xml]: Done

**Comment by davidstap** [comment]
I fixed the names now. Seems the `ingest_tsv.py` script has changed since the last time I've ingested. I should have checked names, sorry.

File: data/xml/2018.eamt.xml (hunk 3 above)
Comment (davidstap) [reply data/xml/2018.eamt.xml]: Fixed the names.

**Comment by davidstap** [comment]
Bumping this; it's been ready to review for a while.

**Review by mjpost — APPROVED** [review]
Thanks for the bump, and the ingest! Looks good to me.

**Pull request merged**